dockerfiles/platform-gnn-gpu/README.md

```diff
@@ -1,45 +1,100 @@
 
-## Tensorflow 2 with GPU support
+## Tensorflow with GPU support
 
-The new Tensorflow 2 and Nvidia Cuda libraries are not always compatible. It will be ideal if we can use Docker image by published by Tensorflow (Docker Hub tensorflow/tensorflow <version>-gpu) but the base image is Ubuntu 18.04 and we use Ubuntu 20.04. So instead we create the images by addiing Tensorflow 2 and Nvida Cuda, etc libraries on the top of hpccystems/platform-core image.
+### Support Tensorflow on a GPU with Docker
+Docker is the easiest way to run TensorFlow on a GPU since the host machine only requires the NVIDIA® driver (the NVIDIA® CUDA® Toolkit is not required).
+When you create a AKS cluster with GPU support VM size NVIDIA driver is installed on the nodes.
+Run "nvidia-smi" will print the NVIDIA driver version. Our HPCC GNN GPU Docker image, platform-gnn-gpu, will have NVIDIA CUDA Toolkit.
+Run "nvcc --version" will print the version. Beware the compatibility issue. For example CUDA Toolkit 11.3 and earlier will not be compatible with NVIDIA Driver 11.4+.  Reference https://docs.nvidia.com/deploy/cuda-compatibility/index.html for more information.
+Current NCS V3 node has  NVIDIA Driver 11.4.
 
-When preparing the Dockerfile, specially for Tensorflow 2 with GPU support it is important to reference
-Tensorflow and Nvidia Cuda Dockerfiles and Docker images to pick compatible libabries.
+### GPU support in our Azure subscription
 
-### Docker image and Dockerfile Reference
+Our Azure subscription does include VM Size of GPU support. These VM Sizes normally are much more expensive than normal VM Sizes, probaly at least 10 times.  Each these VM Sizes with GPU support have different hardware and may require different driver version such as NVidia CUDA driver. Also the Tensorflow 2 and Nvidia Cuda libraries are not always compatible.
 
-#### Tensforflow
+Generally  some ND, NCv2 or NCv3 family VM Sizes should be supported in our Azure subscription
+For example following are supported: standard_nc6s_v3,standard_nc8as_t4_v3,standard_nv12s_v3,standard_nv24s_v3,standard_nv48s_v3
 
-- Dockerfile:
-  https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/dockerfiles/dockerfiles/gpu.Dockerfile
-  Above Dockeerfile is based https://hub.docker.com/layers/nvidia/cuda/. There are runime, base, etc.
-  For now we just reference https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/ and replace the package version for Ubuntu 20.04
+We current build our Docker Image with NVidia CUDA 11.4 and latest Tensorflow packages. User may need to create their own Docker Images with a different CUDA driver version and Tensorflow version.
 
-- The generated tensorflow/tensorflow:2.2.0-gpu
-  https://hub.docker.com/layers/tensorflow/tensorflow/2.2.0-gpu/images/sha256-3f8f06cdfbc09c54568f191bbc54419b348ecc08dc5e031a53c22c6bba0a252e?context=explore
+When preparing the Dockerfile, specially for Tensorflow 2 with GPU support it is important to reference
```

Comment:
When preparing the Dockerfile, especially for Tensorflow 2 with GPU support it is important to reference